src/main/java/com/zipcodewilmington/looplabs/StringDuplicateDeleter.java

```diff
@@ -1,8 +1,120 @@
 package com.zipcodewilmington.looplabs;
 
+import java.util.Arrays;
+
 /**
  * Created by leon on 1/28/18.
  * @ATTENTION_TO_STUDENTS You are forbidden from modifying the signature of this class.
  */
 public final class StringDuplicateDeleter extends DuplicateDeleter<String> {
-}
+    public StringDuplicateDeleter(String[] intArray) {
+        super(intArray);
+    }
+
+    public static void main (String [] args){
+        String [] aTest = new String[]{"aba", "aba", "bba", "bba", "bba", "bba", "bbb", "bbb"};
+        System.out.println(counter(aTest,"bba"));
+        System.out.println(Arrays.toString((deleteString(aTest,"bbb"))));
```

Comment:
same issue as above. Don't put tests in your main class. Also don't commit them if you are writing stuff like this.